### PR TITLE
fix(pci.projects.workflow): display latest execution date

### PR DESCRIPTION
**Pull request by marie-j**
ref: DTRSD-6224

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/workflow/workflow.routing.js [hunk 1]
```diff
@@ -55,31 +54,27 @@ export default /* @ngInject */ ($stateProvider) => {
...
-                        );
-                        return workflow;
-                      });
-                      workflows = concat(workflows, regionWorkflows);
-                      return regionWorkflows;
-                    })
-                    .catch(() => null),
+              .all(workflows)
+              .then((regionWorkflows) =>
+                filter(
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/workflow/workflow.routing.js]: Is [`compact`](https://lodash.com/docs/4.17.15#compact) can suit your needs?

**Comment by ovh-cds** [comment]
CDS Report installation#8538.0 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#8538.1 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#8538.2 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#8780.0 ✘  
* 
  * Build ✘

**Pull request merged**